README.md

```diff
@@ -2225,6 +2237,127 @@ device.services.ptz.removePreset(params).then((result) => {
 });
 ```
 
+---------------------------------------
+## <a id="OnvifServiceSearch-object">`OnvifServiceSearch` object</a>
+
+This object represents the [ONVIF Search Service](https://www.onvif.org/specs/srv/rsrch/ONVIF-RecordingSearch-Service-Spec.pdf).
+
+### <a id="OnvifServiceSearch-getServiceCapabilities-method">getServiceCapabilities(*[callback]*) method</a>
+
+This method sends a `GetServiceCapabilities` command.
+
+### <a id="OnvifServiceSearch-getRecordingSummary-method">getRecordingSummary(*[callback]*) method</a>
+
+This method sends a `GetRecordingSummary` command.
+
+### <a id="OnvifServiceSearch-findRecordings-method">findRecordings(*params[, callback]*) method</a>
+
+This method sends a `FindRecordings` command. The 1st argument `params` must be a hash object consisting of the properties as follows:
+
+Property        |                              |         | Type    | Required |Description
+:---------------|:-----------------------------|:--------|:--------|:---------|:----------
+`Scope`         |                              |         |Object   | required | scope defines the dataset to consider for this search
++-              | `IncludedSources`            |         | Array   | optional | a list of sources that are included in the scope
++-              | +-                           | `Type`  | String  | optional |
++-              | +-                           | `Token` | String  | required |
++-              | `IncludedRecordings`         |         | Array   | optional | a list of recordings that are included in the scope
++-              | `RecordingInformationFilter` |         | String  | optional | an xpath expression used to specify what recordings to search
++-              | `Extension`                  |         | String  | optional | extension point
+`MaxMatches`    |                              |         | Integer | optional | the search will be completed after this many matches
+`KeepAliveTime` |                              |         | Integer | required | the time the search session will be kept alive after responding to this and subsequent requests
+
+```JavaScript
+let params = {
+    'Scope': {
+        'IncludedSources': [
+            {
+                'Type': 'sourceType',
+                'Token': 'sourceToken'
+            }
+        ],
+        'IncludedRecordings': [
+            'recording1'
+        ],
+        'RecordingInformationFilter': 'filter',
+        'Extension': 'extension',
+    },
+    'MaxMatches': 3,
+    'KeepAliveTime': 100
+};
+
+device.services.ptz.findRecordings(params).then((result) => {
```

Comment:
```suggestion
device.services.search.findRecordings(params).then((result) => {
```

Reply:
Hello, you may be right, I sincerely do not remember much about this one. Anyway I guess it isn't really relevant as the project wasn't updated in the past 4 years..